Mathlib/Analysis/NormedSpace/OperatorNorm/Basic.lean

```diff
@@ -329,19 +329,13 @@ theorem opNorm_comp_le (f : E →SL[σ₁₂] F) : ‖h.comp f‖ ≤ ‖h‖ *
       rw [mul_assoc]
       exact h.le_opNorm_of_le (f.le_opNorm x)⟩
 
-
 /-- Continuous linear maps form a seminormed ring with respect to the operator norm. -/
-instance toSemiNormedRing : SeminormedRing (E →L[𝕜] E) :=
-  { ContinuousLinearMap.toSeminormedAddCommGroup, ContinuousLinearMap.ring with
-    norm_mul := fun f g => opNorm_comp_le f g }
+instance toSeminormedRing : SeminormedRing (E →L[𝕜] E) :=
+  { toSeminormedAddCommGroup, ring with norm_mul_le := opNorm_comp_le }
 
 /-- For a normed space `E`, continuous linear endomorphisms form a normed algebra with
 respect to the operator norm. -/
-instance toNormedAlgebra : NormedAlgebra 𝕜 (E →L[𝕜] E) :=
-  { algebra with
-    norm_smul_le := by
-      intro c f
-      apply opNorm_smul_le c f}
+instance toNormedAlgebra : NormedAlgebra 𝕜 (E →L[𝕜] E) := ⟨opNorm_smul_le⟩
```

Comment:
does
```suggestion
instance toNormedAlgebra : NormedAlgebra 𝕜 (E →L[𝕜] E) where
```
work?

Either way, this change feels unrelated to the rest of the PR.

Reply:
It doesn't work, but `where norm_smul_le := norm_smul_le` works. It's not closely related, it's just a minor golf I noticed while working on the immediately adjacent line.

Reply:
I found a nicer formulation which re-uses fields from earlier instances rather than re-declaring them. Hope that's OK.